openstf/metrics/reporter.py

```diff
@@ -74,7 +74,7 @@ def _make_data_series_figures(self, model: RegressorMixin) -> None:
 
         # Make model predictions
         for data_set in self.input_data_list:
-            model_forecast = model.predict(data_set.iloc[:, 1:])
+            model_forecast = model.predict(data_set.iloc[:, 1:-1])
```

Comment:
This need some explanation? Why is the last column ignored?

Reply:
This column contains the horizon information and is not expected by the model as input, I have added a comment to clarify